conf/config.py

```diff
@@ -204,6 +204,12 @@ class BaseConfiguration(object):
         }
     }
 
+    # repositories that should be searched for unpublished images, specifically because of EUS base images
+    UNPUBLISHED_EXCEPTIONS = [
+        {"registry": "registry.access.redhat.com", "repository": "rhel8-2-els/rhel"},
```

Comment:
Could you please make this default to an empty list but configure this in our internal configuration repo of `freshmaker-playbooks`? This is because this is specific to the internal deployment of Freshmaker and I don't want to update this value upstream and perform a deployment just to update this configuration value.

Reply:
https://github.com/redhat-exd-rebuilds/freshmaker/pull/55 Here is the PR for this.

Reply:
Here is the MR for this change: https://gitlab.cee.redhat.com/ansible-playbooks/freshmaker-playbooks/-/merge_requests/84